internal/secrets/aws_test.go

```
@@ -0,0 +1,134 @@
+	container, err := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
+		ContainerRequest: testcontainers.ContainerRequest{
+			Image:        "ministackorg/ministack:latest",
+			ExposedPorts: []string{"4566/tcp"},
+			WaitingFor:   wait.ForListeningPort("4566/tcp"),
+		},
+		Started: true,
+	})
+	if err != nil {
+		t.Fatalf("failed to start ministack container: %v", err)
+	}
+	defer container.Terminate(ctx)
```

golangci-lint (2.12.2)

[error] Error return value of `container.Terminate` is not checked

(errcheck)